AudibleApi/AudibleApi.csproj

```diff
@@ -39,9 +46,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="Cryptography.js">
```

Comment:
I've been meaning to covert this js since forever but it never felt pressing because, it it ain't broke... Thanks for doing this.